lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1038,21 +1038,45 @@ SPIRVValue *SPIRVModuleImpl::addConstant(SPIRVType *Ty, uint64_t V) {
   return addConstant(new SPIRVConstant(this, Ty, getId(), V));
 }
 
+template <spv::Op OC> uint64_t SPIRVConstantBase<OC>::getValue() const {
+  uint64_t TheValue = static_cast<uint64_t>(Words[0]);
+  if (NumWords > 1)
+    TheValue |= static_cast<uint64_t>(Words[1]) << SpirvWordBitWidth;
```

Comment:
I suggest adding an assertion that `NumWords` is not greater than 2

Reply:
Refactored code. No longer relevant.